### PR TITLE
Feature xlsx

### PR DESCRIPTION
**Pull request by Ximi1970**
Optional XLSX output format support. Automatically detects XlsxWriter module.

### CODE REVIEW

File: KiBOM_CLI.py [hunk 1]
```diff
@@ -47,7 +61,10 @@ def say(*arg):
 
 def isExtensionSupported(filename):
     result = False
-    extensions = [".xml",".csv",".txt",".tsv",".html"]
+    if xlsxwriter_available:
```
Comment (SchrodingersGat) [inline KiBOM_CLI.py]: I would have a default extensions array, and then optionally append '.xlsx' if xlsxwriter_available is True

**Comment by SchrodingersGat** [comment]
Hi @Ximi1970 thanks for adding this feature! Please have a look at my review notes above.

**Comment by Ximi1970** [comment]
Hi @SchrodingersGat, I cleaned up the extension handling and also removed the "missing module" message.

**Comment by SchrodingersGat** [comment]
@Ximi1970 awesome, thanks for the contribution

**Pull request merged**